website/content/release-notes/_index.md

```diff
@@ -3,6 +3,51 @@ title: Release Notes
 weight: 8
 ---
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
```

Comment:
link to sample configs will be great to ease migration from configmap to CR, also I assume major update to [doc](https://metallb.universe.tf/configuration) to reflect he new config